packages/marshal/src/marshal.js

```diff
@@ -236,7 +238,7 @@ export const makeMarshal = (
 
     /**
      * @param {{iface?: string, index: number}} slotData
-     * @returns {Remotable | Promise}
+     * @returns {Passable}
```

Comment:
Why is this return type being loosened? A slot really should decode to only a Remotable or Promise.